### ISSUE TITLE
Get [WinError 6] The handle is invalid in Windows 10

### ISSUE THREAD

**Issue by hazardary** [SeleniumHQ/selenium#5316]
## Meta -
OS:  
Windows 10
Selenium Version:  
3.8.0
Browser:  
PhantomJS
Browser Version:  
2.1

## Expected Behavior -
I wrote a pyqt4=4.11.4 application and use pyinstaller=3.3 to make a windows executable. The application use selenium and PhantomJS to browse website and do stuff. It worked perfectly in windows 7.
## Actual Behavior -
`Traceback (most recent call last):
  File "test.py", line 37, in function
  File "site-packages\selenium\webdriver\phantomjs\webdriver.py", line 52, in __init__
  File "site-packages\selenium\webdriver\common\service.py", line 74, in start
  File "subprocess.py", line 640, in __init__
  File "subprocess.py", line 850, in _get_handles
OSError: [WinError 6] The handle is invalid`

## Steps to reproduce -
```
import sys
from PyQt4 import QtGui
import os
from selenium import webdriver

class Example(QtGui.QWidget):
    def __init__(self):
        super(Example, self).__init__()
        self.initUI()
        
    def initUI(self):
        btn = QtGui.QPushButton('Get html', self)
        btn.clicked.connect(self.function)
        self.edit = QtGui.QPlainTextEdit()
        
        layout = QtGui.QVBoxLayout()
        layout.addWidget(btn)
        layout.addWidget(self.edit)
        self.setLayout(layout)
        
        self.setGeometry(300, 300, 300, 500) 
        self.show()
        
    def function(self):
        url = 'https://www.google.com'
        executable_path = os.getcwd() + '\\phantomjs.exe'
        browser = webdriver.PhantomJS(executable_path=executable_path)
        browser.get(url)
        html = browser.page_source
        browser.quit()
        self.edit.setPlainText(html)
        return
def Start():
    global window
    window = Example()
    window.show()

if __name__ == '__main__':
    sys.stderr=open('err.log','w')
    app=QtGui.QApplication.instance()
    if not app:
       app = QtGui.QApplication(sys.argv)
       
    Start()
    sys.exit(app.exec_())
```

I actually found a workaround based on https://stackoverflow.com/questions/40108816/python-running-as-windows-service-oserror-winerror-6-the-handle-is-invalid. In Line 72 of service.py, at the end add stdin,
`self.process = subprocess.Popen(cmd, env=self.env,
                                            close_fds=platform.system() != 'Windows',
                                            stdout=self.log_file, stderr=self.log_file, stdin=self.log_file)`
since I thought the log file won't mess the package. I guess there should be more appropriate way to add stdin. After this, it works find in windows 10.

**Comment by hazardary** [comment]
Hi,
I think this can be closed. See #4503 . service.py has been fixed in the master files but not in 3.8.0.

**Comment by lmtierney** [comment]
@hazardary `3.8.1` is currently out with this fix
